dockerfile_parse/util.py

```diff
@@ -24,8 +24,11 @@ def b2u(string):
 
 def u2b(string):
     """ unicode to bytes"""
-    if ((PY2 and isinstance(string, unicode)) or
-        ((not PY2) and isinstance(string, str))):
+    # pylint: disable=undefined-variable
+    # pylint: disable=unicode-builtin
+    string_type = unicode if PY2 else str
+
+    if isinstance(string, string_type):
```

Comment:
Why not just simply use `six.text_type`?

Reply:
Wow, I didn't know about this feature, thanks :+1: